docker-compose.yml

```diff
@@ -7,16 +7,27 @@ services:
     environment:
       - POSTGRES_DB=${POSTGRES_DB}
       - POSTGRES_USER=${POSTGRES_USER}
+      - PGUSER=${POSTGRES_USER}
```

Comment:
Wouldn't `PGUSER=${POSTGRES_USER}` make `POSTGRES_USER=${POSTGRES_USER}` redundant you think?

Reply:
No, the bug fix is like that.
I can find you the source, but the 2 are mandatory if I remeber well

Reply:
Here
![image](https://github.com/Security-Tools-Alliance/rengine-ng/assets/159943478/10bd7a37-d40a-45f2-af0b-7142028bffe2)
https://stackoverflow.com/a/76757417

Reply:
Ah okay, thanks.